dashboard/src/features/accessControl/actions.js

```diff
@@ -73,14 +74,27 @@ export default {
   },
 
   submitCertificateForm: data => {
+    const fieldInfo = {}
+    for (let index in subjectFieldOptions) {
+      const option = subjectFieldOptions[index]
+      fieldInfo[option.value] = option
+    }
+
     const body = {
       guardType: 'x509',
-      guardData: {subject: {}},
+      guardData: {},
     }
 
     for (let index in data.subject) {
       const field = data.subject[index]
-      body.guardData.subject[field.key] = field.value
+
+      if (fieldInfo[field.key].array) {
+        const values = body.guardData[field.key] || []
```

Comment:
Agreed; the Dashboard needs to submit a valid format to the SDK. I didn't want to tie the two PRs together.